src/ngraph/frontend/onnx_import/utils/broadcasting.cpp

```diff
@@ -141,21 +130,6 @@ namespace ngraph
 {
     namespace onnx_import
     {
-        NodeVector
-            numpy_style_broadcast_for_binary_operation(const std::shared_ptr<ngraph::Node>& left,
-                                                       const std::shared_ptr<ngraph::Node>& right)
-        {
-            const auto& left_shape = left->get_shape();
-            const auto& right_shape = right->get_shape();
-            const auto& numpy_shapes = get_numpy_broadcast_shape(left_shape, right_shape);
-            auto output_shape = numpy_shapes.at(0);
-            auto left_full_shape = numpy_shapes.at(1);
-            auto right_full_shape = numpy_shapes.at(2);
-
-            return {broadcast_node_numpy_style(left, output_shape, left_full_shape),
-                    broadcast_node_numpy_style(right, output_shape, right_full_shape)};
-        }
-
         NodeVector numpy_style_broadcast(NodeVector inputs)
```

Comment:
That's a leftover from my previous task, but this function should probably take a const reference instead of a copy.